plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanSession.kt

```diff
@@ -268,17 +274,22 @@ class CodeWhispererCodeScanSession(val sessionContext: CodeScanSessionContext) {
         createUploadUrlResponse
     } catch (e: Exception) {
         LOG.error { "Security scan failed. Something went wrong uploading artifacts: ${e.message}" }
```

Comment:
Anti pattern logging. Let's just remove all the Error logs if we're throwing exception anyways.https://stackoverflow.com/questions/6639963/why-is-log-and-throw-considered-an-anti-pattern

Reply:
Sorry just noticed this. These logs were there from beginning. Makes sense and smoother if we remove them now.

Reply:
Here we are throwing exception but this is used only for getting telemetry events. For CX we will show generic messages so its better to log the messages if CX wants to dive deeper.

Reply:
Even CX will have get Exception messages in the logs right?

Reply:
Yeah there is a overall Log error message : https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanSession.kt#L239-L245
but this is same over the file If its required I can remove where ever necessary.

Reply:
We can remove since we got that chance to clear it up.